openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -579,19 +591,19 @@ OpenAssessment.ResponseView.prototype = {
     },
 
     /**
-    Handle if the language selection dropdown has been changed
+    Handle if the executor selection dropdown has been changed
     **/
     handleLanguageSelectionChanged: function(){
-    var language = this.getLanguage();
-    this.updateEditorMode(language);
-    this.handleResponseChanged();
-    var defaulCodes  =  {
-           "Python":"import sys\n" +
+        var language = this.getLanguage();
+        this.updateEditorMode(language);
+        this.handleResponseChanged();
+        var defaultCodes  =  {
+           "python":"import sys\n" +
```

Comment:
we should maintain the language version with default code as well

Reply:
Good idea. I'll add it.

Reply:
As discussed in the sprint meeting, we will implement this feature after higher priority tasks are done.